lib/src/main/java/duke/utils/Storage.java

```diff
@@ -0,0 +1,130 @@
+package duke.utils;
+
+import java.util.*;
```

Comment:
Overall everything looks good, but perhaps can use specific import statements instead of using wildcard imports?